packages/salesforcedx-utils-vscode/src/config/configUtil.ts

```diff
@@ -141,15 +144,9 @@ export class ConfigUtil {
       return;
     }
 
-    const username = await getUsernameFor(defaultDevHubUsernameOrAlias);
+    const username = await AuthUtil.getInstance().getUsername(
+      defaultDevHubUsernameOrAlias
+    );
```

Comment:
FYI this is causing the unit test failure b/c this getUsername returns the provided value if no username is found (kinda dumb).

Reply:
Ugh.  How to resolve this?  It seems like AuthUtil is pretty slim, with only the two kinda weird functions.  Do you think we should remove it entirely and use configUtil.getDefaultDevHubUsernameOrAlias and configUtil.getUsername?  I can look at what updating the tests would look like if we did that...

Reply:
sure.  I was going to suggest adding a param to that function to make it sane, but if it can be easily removed and reuse other locations I'm fine with that.

Reply:
I'm fine with creating a follow up ticket to create a single getUsername function